std_msgs/Header header
float64 uav_count
geometry_msgs/Pose[] uav_positions
geometry_msgs/Twist[] uav_velocities